src/openrct2/paint/map_element/surface.c

```diff
@@ -327,7 +327,7 @@ static uint8 viewport_surface_paint_setup_get_relative_slope(rct_map_element * m
 /**
  * rct2: 0x0065E890, 0x0065E946, 0x0065E9FC, 0x0065EAB2
  */
-static void viewport_surface_smoothen_edge(enum edge edge, struct tile_descriptor self, struct tile_descriptor neighbour)
+static void viewport_surface_smoothen_edge(enum edge _edge, struct tile_descriptor self, struct tile_descriptor neighbour)
```

Comment:
This is against our code convention, underscore prefixes are for static variables or private class fields.